charts/metrics-server/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 
 ## [UNRELEASED]
 
+### Fixed
+
+- Fixed nanny's RoleBinding which contained a hard-coded namespace instead of the helm's release namespace
```

Comment:
```suggestion
- Fixed nanny's RoleBinding which contained a hard-coded namespace instead of the Helm's release namespace. ([#1479](https://github.com/kubernetes-sigs/metrics-server/pull/1479)) _@
the-technat_
```

Reply:
should be updated